tools/scripts/rccl_bw_test.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved.
+# Licensed under the MIT License.
```

Comment:
This is ok, although it is nice when the license itself is included in the file. Compare with the license headers for libraries like [rocblas](https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/rocm-5.5.1/library/src/blas3/rocblas_dgmm.cpp), [rocfft](https://github.com/ROCmSoftwarePlatform/rocFFT/blob/rocm-5.5.1/library/src/compute_scheme.cpp) and [rocprim](https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/rocm-5.5.1/rocprim/include/rocprim/config.hpp) do it. When the file is just marked as "Licensed under the MIT License.", the [licensecheck](https://wiki.debian.org/CopyrightReviewTools#licensecheck) tool that I use is not able to automatically detect the license of the file and it must be manually reviewed.

This a rather minor nitpick. Most files need to be manually checked anyway because NCCL's convention isn't autodetected either. Also, now that rccl has been packaged for Debian, I won't be scanning the entire repo from scratch again anyway. Nevertheless, I thought I'd mention it.